sysutils/clamav-server/Portfile

```diff
@@ -3,115 +3,492 @@
 PortSystem              1.0
 
 name                    clamav-server
-version                 0.98.4
+version                 0.101.2
 homepage                http://www.clamav.net/
 categories              sysutils
 platforms               darwin
 supported_archs         noarch
-maintainers             nomaintainer
-license                 BSD
+maintainers             essandess
```

Comment:
`essandess` refers to `essandess@macports.org`. Instead you should use `@essandess` here. See https://guide.macports.org/#reference.keywords.maintainers.

Reply:
Thanks. Fixed.